projects/noloan/firebase/src/main/java/hasadna/noloan/firebase/DbMessages.java

```diff
@@ -114,7 +111,7 @@ public void notifyListeners(int index, SmsMessage smsMessage, Type type) {
   }
 
   // static function to search a given list
-  public static int findSmsByBody(SmsMessage sms, ArrayList<SmsMessage> list) {
+  public static int findSms(SmsMessage sms, ArrayList<SmsMessage> list) {
     for (int i = 0; i < list.size(); i++) {
       if (list.get(i).getBody().contentEquals(sms.getBody())
           && list.get(i).getSender().contentEquals(sms.getSender())) {
```

Comment:
I don't think you need to have it static.
`DbMessages.getInstance()` is static, and returns an object, you can call other functions from.
?

Reply:
But it's not a class function, You don't use any of the class feature with this function.
That way it is static.

Reply:
>But it's not a class function, You don't use any of the class feature with this function.

I understand.
So if you're still having non-related DB messages functions under `DbMessages`  - How is that different from the current `SmsMessages`?